velox/docs/functions/presto/conversion.rst

```diff
@@ -560,28 +576,28 @@ From TIMESTAMP WITH TIME ZONE
 
 The results depend on whether configuration property `adjust_timestamp_to_session_timezone` is set or not.
 
-If set to true, input timezone is ignored and timestamp is returned as is. For example,
-"1970-01-01 00:00:00.000 America/Los_Angeles" becomes "1970-01-01 00:00:00.000".
-
-Otherwise, timestamp is shifted by the offset of the timezone. For example,
+If set to true, timestamp is shifted by the offset of the timezone. For example,
```

Comment:
Wondering why do we want to make this change? Based on the current logic, only when `adjust_timestamp_to_session_timezone == false`, the adjustment would be made right? See [code ref](https://fburl.com/hy0qw0m7).